packages/protobuf/README.md

```diff
@@ -5,35 +5,61 @@ This package provides the runtime library for the code generator plugin
 
 ## Protocol Buffers for ECMAScript
 
-A complete implementation of [Protocol Buffers](https://developers.google.com/protocol-buffers)
-in TypeScript, suitable for web browsers and Node.js.  
-Learn more at [github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
+A complete implementation of [Protocol Buffers](https://developers.google.com/protocol-buffers) in TypeScript,
+suitable for web browsers and Node.js.
 
+## What are Protocol Buffers?
+The official description of Protocol Buffers states that they are: 
 
-It is a complete implementation of [Protocol Buffers](https://developers.google.com/protocol-buffers) 
-in TypeScript, suitable for web browsers and Node.js.
+> a language-neutral, platform-neutral extensible mechanism for serializing structured data.
```

Comment:
We are stripping this section from @bufbuild/protoc-gen-es, why not here as well?